src/server/pegasus_server_impl.h

```diff
@@ -28,7 +28,7 @@ class meta_store;
 class capacity_unit_calculator;
 class pegasus_server_write;
 
-class pegasus_server_impl : public ::dsn::apps::rrdb_service
+class pegasus_server_impl : public pegasus_read_service
```

Comment:
What about change the name of this class to `pegasus_read_service_impl`?

Reply:
https://github.com/Smityz/pegasus/blob/master/src/server/pegasus_server_impl.cpp#L577
Here are also some codes related to write